performance/font-loading.md

```diff
@@ -0,0 +1,64 @@
+
+- [Graded Browser Support](https://github.com/springernature/frontend-playbook/blob/main/practices/graded-browser-support.md)
+  - Is the implementation supported by all "enhanced" browsers? If not, is there a good justification for selecting it anyway?
+  - Does the implementation work in conjunction with the Cut The Mustard technique?
+- [Progressive Enhancement](https://github.com/springernature/frontend-playbook/blob/main/practices/progressive-enhancement.md)
+  - This is not meant to force you to choose a non-js implementation, but rather for you to ensure you consider what will happen for users in varying circumstances from the beginning.
+- [Our Performance Goals](https://github.com/springernature/frontend-playbook/blob/main/performance/performance-checklist.md)
+  - What impact will the implementation have on the page performance of your site? 
+  
+## Quick Tips
+Here are some quick suggestions for techniques that may prove useful:
+
+### Understand the different user experiences in font loading: FOIT, FOUT and FOIT
+
+- FOIT means _flash of invisible text_. Text on the page is hidden while webfonts are loading, so users will see a page with images but no text. We only show the text when web fonts are loaded.
+- FOUT means _flash of unstyled text_. When web fonts are loading, we show users a system font. When the web font gets loaded, we change the text back to the desired web font.
+- FOFT means _flash of faux text_. Only load the roman character set initially and load the others later for less page render blocking. "Faux" refers to the fact the browser fakes italics and bold before those character sets have loaded.
+
+We believe that FOIT should be avoided as it is the worst experience for the user.
+
+### Understand the different implementations available
+
+We highly recommend reading [Zach Leat's comprehensive guide](https://www.zachleat.com/web/comprehensive-webfonts/) to get a good understanding of the different techniques available. There is also a good [follow-up article](https://css-tricks.com/the-best-font-loading-strategies-and-how-to-execute-them/) that summarises Zach's findings.
+
+### Serve WOFF and WOFF2 font formats
+
+WOFF2 has the superior compression. However, unfortunately [it does not cover all of our Enhanced browsers](https://caniuse.com/woff2) at the time of writing. Therefore consider serving both formats. 
+
+### [font-display CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display)
+
+[`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) is useful and has the benefit of being a technique that does not require JavaScript and is easy and straight forward to implement.
+
+`font-display: swap` can be used to achieve FOUT which ensures a user can read text with a fallback font whilst a webfont loads.
```

Comment:
I know this paragraph is not saying that `font-display: swap` is the best or the only useful value for `font-display`, but given that it's the only one mentioned, I feel like we may want to clarify that all possible `font-display` values have serious downsides. Although we're risking this PR becoming comically large if we include that.

For example, `swap` (and `block` too, I think) will cause the whole page to be rendered completely twice, doubling the amount of paint events and increasing. The geometry is probably not the same between fonts, so the shifts in the layout may be annoying or jarring for some people.

Gov.uk uses `fallback`, and, there's an excellent article from Matt Hobbs showing the whole decision process of why they decided to use that value:
https://calendar.perfplanet.com/2020/a-font-display-setting-for-slow-connections/

I'm not saying that `fallback` is best by the way, just that this is a bit too complex for a short mention so maybe we can just let others do all the hard work and link to that post, or a similar one?